plugins/lookup/etcd3.py

```diff
@@ -0,0 +1,230 @@
+# -*- coding: utf-8 -*-
+#
+# (c) 2020, SCC France, Eric Belhomme <ebelhomme@fr.scc.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    author:
+    - Eric Belhomme <ebelhomme@fr.scc.com>
+    lookup: etcd3
+    short_description: get info from an etcd3 server
+    description:
+    - Retrieves data from an etcd3 server.
+
+    options:
+        _terms:
+            description:
+            - The list of keys to look up on the etcd3 server.
+            type: list
+            elements: str
+            required: True
+        prefix:
+            description:
+            - look for key or prefix key
```

Comment:
```suggestion
            - Look for key or prefix key.
```
As the description above. Also, please adjust all other descriptions below in the same way.

Reply:
I updateds some 'description' fields, and checked all for pontuation. I also checked on on etcd3 project pages on how they use upper/lower case for etcd3 naming, and they **always** use lowercase. So I do the same.